scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -51,24 +52,49 @@ class ExperimentalScanner(
     val storageWriters: List<ScanStorageWriter>,
     val packageProvenanceResolver: PackageProvenanceResolver,
     val nestedProvenanceResolver: NestedProvenanceResolver,
-    val scannerWrappers: List<ScannerWrapper>
+    val scannerWrappers: Map<DataEntity, List<ScannerWrapper>>
 ) {
     init {
-        require(scannerWrappers.isNotEmpty()) {
+        require(scannerWrappers.isNotEmpty() && scannerWrappers.any { it.value.isNotEmpty() }) {
             "At least one ScannerWrapper must be provided."
         }
     }
 
     suspend fun scan(ortResult: OrtResult): OrtResult {
         val startTime = Instant.now()
 
-        val result = scan(
-            (ortResult.getProjects().map { it.toPackage() } + ortResult.getPackages().map { it.pkg }).toSet()
-        )
+        val projectScannerWrappers = scannerWrappers[DataEntity.PROJECTS].orEmpty()
+        val packageScannerWrappers = scannerWrappers[DataEntity.PACKAGES].orEmpty()
+
+        val projectResults = if (projectScannerWrappers.isNotEmpty()) {
+            val packages = ortResult.getProjects().mapTo(mutableSetOf()) { it.toPackage() }
+
+            log.info { "Scanning ${packages.size} projects with ${projectScannerWrappers.size} scanners." }
+
+            scan(packages, DataEntity.PROJECTS)
+        } else {
+            log.info { "Skipping scan of projects because no project scanner wrapper is configured." }
```

Comment:
Would it make sense to drop the term "wrapper" from the log messages, because it's a technical detail?